src/WaveTrack.h

```diff
@@ -458,6 +458,9 @@ class AUDACITY_DLL_API WaveTrack final : public PlayableTrack {
    // to the newly created clip.
    WaveClip* CreateClip();
 
+   // Same as ::CreateClip() but also sets an offset of the new clip
+   WaveClip* CreateClip(double offset);
```

Comment:
WaveClip::mOffset defaults to 0 in the constructor that doesn't copy from another clip.

So you can combine the two overloads of CreateClip into one, defaulting offset to 0, and behavior will be the same.  Eliminating the duplication of code would be preferable.